introduction.md

```diff
@@ -1,2 +1,7 @@
-Hi, my name is toprahmin. Thanks for having me at the dojo. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?